src/main/java/com/coursesPlatform/coursePortfolio/Offer.java

```diff
@@ -0,0 +1,74 @@
+package com.coursesPlatform.coursePortfolio;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+class Offer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @OneToOne
+    private Category category;
```

Comment:
I think it should be possible to avoid adding Category and Course here. It should be possible to take those data from lectures.